.github/workflows/deploy-staging.yaml

```diff
@@ -246,64 +245,20 @@ jobs:
       - id: deploy-template-cognito
         name: Deploy CloudFormation stack for Cognito pool clients
         uses: aws-actions/aws-cloudformation-github-deploy@v1
-        if: ${{ github.ref == 'refs/heads/master' }}
         with:
           parameter-overrides: "Environment=staging,SandboxID=${{ github.event.inputs.sandbox-id }}"
           name: ${{ steps.set-stack-name.outputs.cognito-name }}
           template: cf/userpoolclient.yaml
           no-fail-on-empty-changeset: "1"
           capabilities: "CAPABILITY_IAM,CAPABILITY_NAMED_IAM"
 
-      - id: disable-admin-enforcement
-        name: Temporarily disable admin enforcement
-        uses: benjefferies/branch-protection-bot@1.0.7
-        with:
-          access_token: ${{ secrets.API_TOKEN_GITHUB }}
-          owner: ${{ github.repository_owner }}
-          repo: iac
-          enforce_admins: false
-          retries: 8
-
-      - id: pull
-        name: Pull and allow unrelated histories.
-        run: |-
-          git pull --allow-unrelated-histories
-
-      - id: push-deployment
-        uses: stefanzweifel/git-auto-commit-action@v4
-        with:
-          commit_message: Deploy staging environment ${{ github.event.inputs.sandbox-id }}
-
-      - id: on-failure-retry
-        if: failure()
-        name: Retry in case push failed.
-        uses: nick-invision/retry@v2
-        with:
-          timeout_seconds: 30
-          max_attempts: 5
-          retry_on: error
-          command: git config user.name "GitHub Actions" && git pull --allow-unrelated-histories --no-edit && git push origin master
-          # Add jitter to break up correlated events.
-          on_retry_command: sleep $((5 + RANDOM % 10))
-
-      - id: enable-admin-enforcement
-        name: Re-enable admin enforcement
-        uses: benjefferies/branch-protection-bot@1.0.7
-        if: always()
-        with:
-          access_token: ${{ secrets.API_TOKEN_GITHUB }}
-          owner: ${{ github.repository_owner }}
-          repo: iac
-          enforce_admins: true
-          retries: 8
-
-
-      - name: Push staging deployment template to releases
```

Comment:
all this code was here to handle the errors due to the repo being modified in-between the initial checkout and the modification and push at this step. I think the same might happen here as we allow two staging envs to be deployed at the same time but I'm not sure. We can leave this removed but it'll need to be monitored in case the errors start showing up.